modules/RTC/TraceablePeerConnection.js

```diff
@@ -1883,6 +1885,45 @@ TraceablePeerConnection.prototype._insertUnifiedPlanSimulcastReceive
         });
     };
 
+/**
+ * Sets the max bitrate on the RTCRtpSender so that the
+ * bitrate of the enocder doesn't exceed the configured value.
+ * This is needed for the desktop share until spec-complaint
+ * simulcast is implemented.
+ * @param {JitsiLocalTrack} localTrack - the local track whose
+ * max bitrate is to be configured.
+ */
+TraceablePeerConnection.prototype.setMaxBitRate = function(localTrack) {
+    const mediaType = localTrack.type;
+
+    if (mediaType !== MediaType.AUDIO) {
+        if (!this.peerconnection.getSenders) {
+            logger.debug('Browser doesn\'t support RTCRtpSender');
+
+            return;
+        }
+        const videoType = localTrack.videoType;
+        const trackId = localTrack.track.id;
+
+        this.peerconnection.getSenders()
+            .filter(s => s.track && s.track.id === trackId)
+            .forEach(sender => {
+                // eslint-disable-next-line prefer-const
+                let parameters = sender.getParameters();
+
+                if (parameters.encodings && parameters.encodings.length) {
+                    logger.info('Setting max bitrate on video stream');
+                    parameters.encodings.forEach((encoding, idx) => {
+                        encoding.maxBitrate = videoType === 'desktop'
```

Comment:
does it make sense to check if `maxBitrate` is supported?

Reply:
maxBitrate property in the encoding object is not usually set the first time when the encoder is created so I can't check for that. Maybe I can check if it is actually set after it is applied, will that suffice ? maxBitrate is supported from Chrome 69 onwards.